solr/core/src/test/org/apache/solr/cli/PostToolTest.java

```diff
@@ -85,8 +85,10 @@ public void testBasicRun() throws Exception {
 
     String[] args = {
       "post",
-      "--solr-update-url",
-      cluster.getJettySolrRunner(0).getBaseUrl() + "/" + collection + "/update",
+      "--solr-url",
+      cluster.getJettySolrRunner(0).getBaseUrl().toString(),
```

Comment:
`getBaseUrl()` extends v1 API path `/solr`, and the new logic does also add `/solr/` if `--solr-url` is used. This results to a path like `/solr/solr/collection/update`.

Reply:
i think I fixed it with the use of `SolrCLI.normalizeSolrUrl() in the tool`